tests/test_browser_custom_keybinds.py

```
@@ -13,14 +13,51 @@
   - Debug socket commands enabled (`set_shortcut`, `simulate_shortcut`)
 """
 
+import json
 import os
 import sys
 import time
-from typing import Optional
+from typing import Any, Dict, Optional
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)